src/main/java/seedu/address/commons/core/index/Index.java

```diff
@@ -51,4 +51,15 @@ public boolean equals(Object other) {
                 || (other instanceof Index // instanceof handles nulls
                 && zeroBasedIndex == ((Index) other).zeroBasedIndex); // state check
     }
+
+    @Override
+    public int compareTo(Index index) {
+        if (this.zeroBasedIndex > index.zeroBasedIndex) {
+            return 1;
+        } else if (this.zeroBasedIndex == index.zeroBasedIndex) {
+            return 0;
+        } else {
+            return -1;
```

Comment:
This can be shortened to just this.zeroBasedIndex - index.zeroBasedIndex. (I think)